graphistry/Plottable.py

```diff
@@ -580,15 +580,15 @@ def infer_labels(self) -> 'Plottable':
         ...
 
     
-    @overload
+    @overload  # type: ignore[overload-overlap]
```

Comment:
should we even have overload for these? bc i'd expect the signatures to match, right?